2008-04-06  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/net/ethernet.h: Add defines for 
	ETHERTYPE_VLAN and ETHERTYPE_IPV6.

diff --git a/sysdeps/unix/sysv/linux/net/ethernet.h b/sysdeps/unix/sysv/linux/net/ethernet.h
index 7ca8e83..ff9f506 100644
--- a/sysdeps/unix/sysv/linux/net/ethernet.h
+++ b/sysdeps/unix/sysv/linux/net/ethernet.h
@@ -48,6 +48,8 @@ struct ether_header
 #define	ETHERTYPE_IP		0x0800		/* IP */
 #define	ETHERTYPE_ARP		0x0806		/* Address resolution */
 #define	ETHERTYPE_REVARP	0x8035		/* Reverse ARP */
+#define	ETHERTYPE_VLAN		0x8100		/* IEEE 802.1Q VLAN tagging */
+#define	ETHERTYPE_IPV6		0x86dd		/* IP protocol version 6 */
 
 #define	ETHER_ADDR_LEN	ETH_ALEN                 /* size of ethernet addr */
 #define	ETHER_TYPE_LEN	2                        /* bytes in type field */
